### DISCUSSION TITLE
AI失準

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#624] [General]
使用AI的同時，我們應該如何注意倫理之類的問題，讓社會總體的福祉達到最大呢?
https://www.ithome.com.tw/news/155024

**Comment by awehao** [comment]
"臉部辨識AI失準警察抓錯人，美國無辜黑人男子入獄一星期"，這是裡面的新聞標題，我覺得行走江湖還是要相信自己，不能過於仰賴科技或各種手段，科技只是輔助，科技領進門，使用在個人==

**Comment by awkward-willy** [comment]
在未完全成熟的階段，或許 AI 更適合作為輔助人們判斷的工具，仍仰賴專業人員做出合適的判斷。同時這樣的議題也能帶出背後更深、更值得探討的倫理問題，例如：這樣的失誤應向何者究責，是開發軟體的工程師？誤判的專業人員？抑或是系統本身應受到短暫禁止？這些都是值得探討的問題。

**Comment by manipulateed** [comment]
AI的結果只能說是參考，人實際去比對或是查證才是最重要的，而且就算抓到無辜之人，也應該作出解釋吧，不然那些精神損失或是當中有拷問的傷害，又應當給予什麼樣的交代。

**Comment by larry40305** [comment]
我覺得有沒有AI應該沒有差很多，畢竟美國警察不盡職亂抓人的新聞也不少。
